### ISSUE TITLE
Connection Target Error

### ISSUE THREAD

**Issue by Geonjae-Joo** [eclipse/packages#202]
https://github.com/eclipse/ditto/issues/971

Ditto can't publish command any more as I updated packages from 0.1.2 to 0.1.3. Is it because of COMMAND ROUTER which is updated recently in HONO??
Please help me out! Thanks

**Comment by calohmn** [comment]
Thanks for reporting. I could reproduce the issue and this is a bug in the 0.1.3 chart. Until the chart version with the fix (#203) gets published, you can add these parameters when deploying the chart as a workaround:
````
--set hono.deviceConnectionService.enabled=true --set hono.dataGridExample.enabled=true
````
(This will deploy an Infinispan data grid for storage of the device connection data. The updated cloud2edge chart will use an embedded cache instead, requiring less resources. For a production-grade deployment, the data grid option should be used.)

**Comment by Geonjae-Joo** [comment]
@calohmn  Thanks you so much!!! it is working!!!!
I've been struggling for quite a time.
